crates/starknet-devnet/src/main.rs

```diff
@@ -168,6 +171,70 @@ async fn main() -> Result<(), anyhow::Error> {
         let json_rpc_client = JsonRpcClient::new(HttpTransport::new(url.clone()));
         set_and_log_fork_config(&mut starknet_config.fork_config, &json_rpc_client).await?;
 
+        let block_id = BlockId::Number(
```

Comment:
the only (stylistic) comment  I can make is to maybe considering extracting some of this logic in a fn outside main()

Reply:
https://github.com/0xSpaceShard/starknet-devnet-rs/pull/590/commits/29a2dce42fb48dcdda5ffc5ffff68621a7675d00